Agreements/Internal/36_6.txt

```diff
@@ -1 +1,19 @@
 COPX:*:*:*:EGBB:*:London S36:London S6:*:31000:|
+COPX:*:*:*:EGLL:*:London S36:London S6:*:27000:|
```

Comment:
I'm assuming you have a source for this being 270 and not 310?

Reply:
@cpawley  - Not sure 270 makes sense if S36's base is still 305. S6 would be giving them 270 GIBSO to S20.
Presumably it's still descending 310 from 36 to 6?

Reply:
```suggestion
COPX:*:*:*:EGLL:*:London S36:London S6:*:31000:|
```